templates/embeds/card.html.twig

```diff
@@ -28,15 +30,15 @@
                 {% if _collapsible %}
                     {{ button.action_collapsebutton(
                         collapsible_title|default('Toggle visibility'|trans({}, 'TablerBundle')),
-                        '#' ~ _id
+                        '.' ~ _collapsible_class
                     ) }}
                 {% endif %}
             </div>
         {% endif %}
     </div>
     {% endif %}
     {% if block('box_header_after') is defined %}{{ block('box_header_after') }}{% endif %}
-    <div {% if _collapsible %}id="{{ _id }}" {% endif %}class="{% if _collapsible %}{% if _collapsed %} collapse{% else %} show{% endif %}{% endif %} card-body{% if block('box_body_class') is defined %} {{ block('box_body_class') }}{% endif %}{% if _fullsize %} p-0{% endif %}">{{ block('box_body') }}</div>
```

Comment:
It's only used for body id and only that.
Not used for collaspe event/action

Reply:
@kevinpapst after using it, `id` is more logic to have on the `card` and not the `card-body`.
With the current state, targeting changes by the id, does not affect the footer.

Reply:
👍